app/test_main.py

```diff
@@ -1,3 +1,41 @@
+import pytest
+from typing import List
 from app.main import get_human_age
```

Comment:
those are different types of imports. Add blank lines where it is needed according to PEP8